.gitlab-ci.yml

```diff
@@ -88,12 +88,13 @@ release_integration:
     - git clean -df
     - git reset --hard
     - for i in $(seq 1 40); do
-    -   if [[ "pending" != $(scripts/status.sh HumanCellAtlas dcp integration) ]]; then break; fi
+    -   status=$(scripts/status.sh HumanCellAtlas dcp integration)
+    -   if [[ "pending" != ${status} && "running" != ${status} ]]; then break; fi
```

Comment:
Technically, "pending" and "running" don't need to be quoted, but `${status}` could benefit from being quoted.

Reply:
fixed